### PR TITLE
test: add more continuous logging unit tests in swift

### PR DESCRIPTION
**Pull request by jayahariv**
* check for maxSize
* check for disabled logging
* check for re-enabled logging

link: #2302

### CODE REVIEW

File: Swift/Tests/LogTest.swift [hunk 1]
```diff
@@ -46,6 +46,44 @@ class LogTest: CBLTestCase {
                          maxRotateCount: Database.log.file.maxRotateCount)
     }
     
+    func writeOneKiloByteOfLog() {
+        let message = "11223344556677889900"
+        for _ in 0..<23 {
+            Log.log(domain: .database, level: .verbose, message: message)
```
[the remark this answers is not in the document]
Comment (jayahariv) [reply Swift/Tests/LogTest.swift]: wil check this

File: Swift/Tests/LogTest.swift [hunk 2]
```diff
@@ -189,6 +222,73 @@ class LogTest: CBLTestCase {
...
+        Database.log.file.maxSize = 1024
+        Database.log.file.level = .verbose
+        
+        writeOneKiloByteOfLog()
+        writeOneKiloByteOfLog()
+        
+        var totalFilesInDirectory = (Database.log.file.maxRotateCount + 1) * 4
+        
+        #if DEBUG
+        totalFilesInDirectory = totalFilesInDirectory + 1
```
Comment (jayahariv) [inline Swift/Tests/LogTest.swift]: wrong!

Comment (jayahariv) [reply Swift/Tests/LogTest.swift]: need to fix this if the debug is not writing to logs!

File: Swift/Tests/LogTest.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (jayahariv) [reply Swift/Tests/LogTest.swift]: I was setting the .verbose and that was not printing the debug logs.

[the remark this answers is not in the document]
Comment (jayahariv) [reply Swift/Tests/LogTest.swift]: this is resolved!! marking it as resolved is not working! slow GH

File: Swift/Tests/LogTest.swift [hunk 3]
```diff
@@ -46,6 +48,43 @@ class LogTest: CBLTestCase {
                          maxRotateCount: Database.log.file.maxRotateCount)
     }
     
+    func writeOneKiloByteOfLog() {
+        /// add a second delay to skip the filesystem flush
+        Thread.sleep(forTimeInterval: 1.0)
```
Comment (pasin) [inline Swift/Tests/LogTest.swift]: Sound like this should be moved to after `writeAllLogs("123456789")`?

Comment (jayahariv) [reply Swift/Tests/LogTest.swift]: i will check this and get more context on this

**Comment by jayahariv** [comment]
Closing this PR, since the file logging is going to change.

**Pull request closed without merging**